### ISSUE TITLE
Problems running Sails.js tests in Windows

### ISSUE THREAD

**Issue by amyspark** [balderdashy/sails#3331]
Version: 0.12.0-rc3 (as of 43b3717343ea6098020d9fa4e617089bb62daf32)
Windows: 7
Node: 4.2.1
npm: 3.3.10
Running inside NetBeans IDE Build 201510060002

I am unable to run the test suite included with Sails due to the following errors:

- In order to execute sails successfully, CMD needs to include `node` in the command line (just `./bin/sails.js` will not work).
    ```javascript
not ok 53 API and adapter generators "before all" hook
Error: Error: Command failed: C:\Windows\system32\cmd.exe /s /c "./bin/sails.js new testApp"
"." no se reconoce como un comando interno o externo,

programa o archivo por lotes ejecutable.


at D:\Documents\Amyspark\sails\test\integration\generate.test.js:26:18
at ChildProcess.exithandler (child_process.js:210:5)
at maybeClose (internal/child_process.js:818:16)
at Socket.<anonymous> (internal/child_process.js:319:11)
at Pipe._onclose (net.js:469:12)
not ok 53 globals ::  with default settings "before all" hook, suite=globals ::  with default settings, testcase="before all" hook, duration=undefined
Error: ENOENT: no such file or directory, scandir 'D:\Documents\Amyspark\test\integration\fixtures\sampleapp'
at Error (native)
at Object.fs.readdirSync (fs.js:813:18)
at D:\Documents\Amyspark\sails\test\integration\helpers\appHelper.js:58:25
at ChildProcess.exithandler (child_process.js:194:7)
at maybeClose (internal/child_process.js:818:16)
at Process.ChildProcess._handle.onexit (internal/child_process.js:211:5)

Done.

    ```

- Fixing the above error with a `exec(sailsBin` -> `exec('node ' + sailsBin`, another error pops up. It would seem that Sails cannot find the `sails-disk` adapter, even though it's been installed and `npm link`ed.
    ```bat
  globals ::
    with default settings
error: Trying to use unknown adapter, "sails-disk", in model `empty`.
error: Are you sure that adapter is installed in this Sails app?
error: If you wrote a custom adapter with identity="sails-disk", it should be in this app's adapters
 directory.
error: Otherwise, if you're trying to use an adapter named `sails-disk`, please run `npm install sai
ls-disk --save
    ```

**Comment by sailsbot** [comment]
Thanks for posting, @amyspark.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
